core/templates/dev/head/css/oppia.css

```diff
@@ -3951,3 +3951,10 @@ span.oppia-issues-learner-action {
   width: 100%;
   border: 1px solid black;
 }
+
+/* To fix the shifting of tooltip when text is too long. */
+
```

Comment:
Remove this line so there is no space between the comment and the CSS rule

Reply:
Sure @vojtechjelinek

Reply:
Removed the line. Kindly merge. @vojtechjelinek